kombuworker/log.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import time
+import os
+import pathlib
+
```

Comment:
same question here about splitting the standard library stuff - is this about standard `import`s vs. `from ~~ import ~~`?